community/knative/charts/build/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+appVersion: "0.2.0"
+description: A Helm chart for Kubernetes
+name: build
+version: 0.1.0
+icon: https://raw.githubusercontent.com/knative/docs/master/images/logo/rgb/knative-logo-rgb.jpg
+kubeVersion: ">=1.10.3"
+tillerVersion: ">=2.10.0"
```

Comment:
Needs to include at least 2.9.1 for support on 3.1.2

Reply:
This isn't ideal as the new tiller version allowed us to utilise new features that streamline CRD installs. 
However, as that is not allowed/recommended on ICP, we will downgrade tiller requirements for now and make CRD installs a pre-req step that users will do manually before installing the chart.